app/models/invenio_rdm_record_converter.rb

```diff
@@ -247,9 +255,9 @@ def build_creator_contributor_json(creator)
         }
       }
 
-      if dh_user.orcid.present?
+      if dh_user&.orcid.present?
```

Comment:
My bad with that `|| creator.include?(",")` refactor.

Reply:
Nah, that's my bad for not noticing lol. I wrote this myself after all.